app-modules/audit/src/Overrides/Concerns/AttachOverrides.php

```diff
@@ -99,7 +104,13 @@ public function detach($ids = null, $touch = true)
         return empty($results) ? 0 : $results;
     }
 
-    public function sync($ids, $detaching = true)
+    /**
+     * @param array<string, mixed> $ids
+    * @param mixed $detaching
```

Comment:
Some weird spacing on the first one here. Also it looks like `$detaching` is a boolean, not mixed right?